pkg/api/handlers/libpod/manifests.go

```diff
@@ -460,6 +461,20 @@ func ManifestModify(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(body.ManifestAddOptions.Annotation) != 0 {
```

Comment:
Can you throw an error if Annotation and Annotations are both set?

Reply:
Done.